src/it/saxon/src/test/java/com/jcabi/saxon/SaxonSampleTest.java

```diff
@@ -131,7 +144,13 @@ public Void call() throws Exception {
                 );
                 return null;
             }
-        } .call();
+        };
+        final ExecutorService executorService = Executors.newFixedThreadPool(5);
+        for (int count = zero; count < fifty; count = count + 1) {
+            executorService.submit(callable);
+        }
+        executorService.awaitTermination(ten, TimeUnit.SECONDS);
+        executorService.shutdown();
```

Comment:
@tmjee it's better to do `shutdownNow` here. If some task will take forever then we never finish.